internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/controller/TestAdminToolEndToEnd.java

```diff
@@ -154,7 +154,7 @@ public void testNodeReplicasReadinessCommand() throws Exception {
     VeniceServerWrapper server = venice.getVeniceServers().get(0);
     String[] nodeReplicasReadinessArgs =
         { "--node-replicas-readiness", "--url", venice.getLeaderVeniceController().getControllerUrl(), "--cluster",
-            clusterName, "--storage-node", Utils.getHelixNodeIdentifier(server.getPort()) };
+            clusterName, "--storage-node", Utils.getHelixNodeIdentifier(Utils.getHostName(), server.getPort()) };
```

Comment:
Since a lot of the test code ends up passing `Utils.getHostName()`, I wonder if we should have two overloaded versions of `Utils::getHostName`? We could keep one with the same signature as today which passes `Utils.getHostName()` into the other one? Not a big deal though. LMK what you think.

Reply:
Sorry, I meant two overloaded versions of `Utils:: getHelixNodeIdentifier`, indeed. But I don't mind adopting your change as is also. It was just a minor point.